public/app/features/alerting/unified/utils/amroutes.ts

```diff
@@ -130,24 +135,38 @@ export const formAmRouteToAmRoute = (
   id2ExistingRoute: Record<string, Route>
 ): Route => {
   const existing: Route | undefined = id2ExistingRoute[formAmRoute.id];
+
+  const {
+    overrideTimings,
+    groupWaitValue,
+    groupWaitValueType,
+    groupIntervalValue,
+    groupIntervalValueType,
+    repeatIntervalValue,
+    repeatIntervalValueType,
+  } = formAmRoute;
+
+  const overrideGroupWait = overrideTimings && groupWaitValue;
```

Comment:
Then we could create a method that converts to and from string

Reply:
Ideally yes, though the reason they are separate props here is because the form's model (where these are individual fields) is bleeding through to the data structures here 😅